### PR TITLE
feat: Added configuration options for probes

### PR DESCRIPTION
**Pull request by Linuem**
* Added the configuration options for all possible values of liveness
and readiness probes (initialDelaySeconds, periodSeconds,
timeoutSeconds, successThreshold, failureThreshold)

### CODE REVIEW

**Comment by Linuem** [comment]
Hi there,

I added some of these options as our k8s service gets 503 sometimes due to the liveness check. This happens when getting a big amount of packages. Therefor I needed the config options to change the timeouts.

Regards,
Sebastian

**Comment by Ashtonian** [comment]
Running into the same issue.

**Comment by pshanoop** [comment]
@juanpicado what happened to the linter? Can you retrigger?

**Comment by juanpicado** [comment]
I will double check, I haven't touch this repo since a while.

**Comment by juanpicado** [comment]
Error says
````
w chart version: 4.6.2
------------------------------------------------------------------------------------------------------------------------
 ✖︎ verdaccio => (version: "4.6.2", path: "charts/verdaccio") > Chart version not ok. Needs a version bump!
------------------------------------------------------------------------------------------------------------------------
Error linting charts
````

You forgot bump up :)

**Comment by pshanoop** [comment]
> Error says
> 
> ```
> w chart version: 4.6.2
> ------------------------------------------------------------------------------------------------------------------------
>  ✖︎ verdaccio => (version: "4.6.2", path: "charts/verdaccio") > Chart version not ok. Needs a version bump!
> ------------------------------------------------------------------------------------------------------------------------
> Error linting charts
> ```
> 
> You forgot bump up :)

huh, I can't see these logs. 

> lint-test Expected — Waiting for status to be reported
Just this.

**Comment by juanpicado** [comment]
I will check your permissions, you should be able

**Comment by juanpicado** [comment]
@pshanoop you are maintainer now

File: charts/verdaccio/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A lightweight private node.js proxy registry
 name: verdaccio
-version: 4.6.2
+version: 4.6.3
```
Comment (juanpicado) [inline charts/verdaccio/Chart.yaml:4]: 🤔 ok, weird, checks are a bit confused now, maybe is a GH issue.

**Review by juanpicado — APPROVED** [review]
LGTM

**Comment by pshanoop** [comment]
> I will check your permissions, you should be able

Seems I lost approval rights too.

**Comment by juanpicado** [comment]
> > I will check your permissions, you should be able
> 
> 
> 
> Seems I lost approval rights too.

Ok that's odd, you have maintainer role assigned. Could you check if you have any invitation pending to be accepted?

**Review by rblaine95 — APPROVED** [review]
LGTM 👍

**Comment by Linuem** [comment]
Thank you for merging the PR <3

**Pull request merged**